src/azure-cli/setup.py

```diff
@@ -131,7 +131,7 @@
     'azure-synapse-accesscontrol~=0.2.0',
     'azure-synapse-artifacts~=0.3.0',
     'azure-synapse-spark~=0.2.0',
-    'cryptography>=2.3.1,<3.0.0',
+    'cryptography>=3.0,<4.0',
```

Comment:
hi @fengzhou-msft to avoid any breaking, is it ok to ping to exact version? at least >=3.2, where the security alert got fixed.